crypto/signing/mcl/scalar.go

```diff
@@ -0,0 +1,257 @@
+package mcl
+
+import (
+	"crypto/cipher"
+
+	"github.com/ElrondNetwork/elrond-go/core/check"
+
+	"github.com/ElrondNetwork/elrond-go/crypto"
+	"github.com/ElrondNetwork/elrond-go/crypto/signing/mcl/bls-go-binary/bls"
+)
+
+// MclScalar -
+type MclScalar struct {
```

Comment:
Could be just scalar since it's in the `mcl` package. Also that will eliminate the weak warning